src/components/create.vue

```diff
@@ -16,6 +16,26 @@
                 required=""
                 @focus="s1c=''"
             >
+            <p class="my-3 font-weight-bold">{{ $t('chain_cta') }}</p>
```

Comment:
Is that a specific approach to handle classes due to the framework? 
It seems off as it introduces in-code styling

Reply:
Those are templates...they're supposed to include styling , unless I'm misunderstanding the question

Reply:
For example `font-weight-bold`:

You already define in the code that you want bold face, the CSS only decides now how it realizes bold face. IMO the CSS class should identify which component it is you are styling, and decide in the CSS how it is styled (maybe a new theme would like italic face instead of bold).

In BitShares UI we try to follow BEM CSS guidelines that @gibbsfromncis uses

Reply:
ah...you're suggesting that I abstract class names so people can theme using CSS only.

However I'm not writing any CSS. I'm using bootstrap and using bootstrap classes for theming (https://getbootstrap.com/docs/4.0/utilities/text/)

Also, not really supposed to allow theming :)